### PR TITLE
Oadp 4074 network settings

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

https://issues.redhat.com/browse/OADP-4074

### Version(s):

OCP 4.13 +

### Link to docs preview:

[OADP-recommended network settings](https://84193--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.html)

### QE review:
- [X] QE has approved this change. See this PR - [HERE](https://github.com/openshift/openshift-docs/pull/82622) 

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
https://github.com/openshift/openshift-docs/pull/82622 was peer-reviewed by @mburke5678

But Vibhuti left Red Hat, a couple of weeks back, so I have made the changes requested and would like to push to merge review

**Comment by mburke5678** [comment]
Peer review in https://github.com/openshift/openshift-docs/pull/82622.

**Comment by mburke5678** [comment]
@anarnold97 Please squash the commits.

**Comment by anarnold97** [comment]
@mburke5678 - sorry, that was an idiot thing

**Comment by anarnold97** [comment]
@mburke5678 - thanks for doing this. do you want me to get to the cherry pick/fixes for 4.13 later this evening as yours seems to have a problem?

File: backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-recommended-network-settings.adoc"]
```
Comment (kalexand-rh) [inline backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.adoc:2]: @anarnold97, the ".adoc" part of this ID is making the PV1 builds fail. Will you open a new PR to remove it?

Comment (mburke5678) [reply backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.adoc:2]: @kalexand-rh  PTAL https://github.com/openshift/openshift-docs/pull/84386

Comment (anarnold97) [reply backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.adoc:2]: @kalexand-rh - https://github.com/openshift/openshift-docs/pull/84387

**Pull request merged**